### DISCUSSION TITLE
What are your thought about the new feature "search" on chatgpt ?

### DISCUSSION THREAD

**Discussion by iButcat** [assafelovic/gpt-researcher#977] [General]
I really enjoy using GPT Researcher, as it speeds up my research and is open source. I also use ChatGPT, and I find the new ‘search’ feature effective, even though its scope seems somewhat limited. What are your thoughts on this? Do you think it could achieve the same level of research capabilities as GPT Researcher?

On my side, I think it may not match the depth and specialized research capabilities of GPT Researcher, especially for highly academic or niche topics. It also provide the source which honestly look more accurate than Chatgpt. 

<img width="979" alt="Screenshot 2024-11-05 at 23 54 21" src="https://github.com/user-attachments/assets/a87f9095-594b-4cdf-94ef-0c570c7ee9c6">

**Comment by ElishaKay** [comment]
Welcome @iButcat 

Sounds interesting & thanks for the kind words 🙏

I'd be surprised if they're referencing as many sources and preventing hallucinations as well as GPT-Researcher, but I'm happy to be proven wrong

**Comment by coreylansdell** [comment]
I can't get it working correctly. I seem to be missing the newest features and it just seems to lock up on me.. 
![Screenshot 2024-11-15 at 10 28 16 PM](https://github.com/user-attachments/assets/4b072ae6-ad81-4638-8c53-4ecdc69e7058)
